wp-content/civi-extensions/goonjcustom/api/v3/Goonjcustom/InstitutionCollectionCampCron.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ */
+
+use Civi\Api4\EckEntity;
+use Civi\Api4\OptionValue;
+use Civi\InstitutionCollectionCampService;
+
+/**
+ * Goonjcustom.CollectionCampCron API specification (optional)
+ * This is used for documentation and validation.
+ *
+ * @param array $spec
+ *   description of fields supported by this API call.
+ *
+ * @return void
+ */
+function _civicrm_api3_goonjcustom_institution_collection_camp_cron_spec(&$spec) {
+  // There are no parameters for the Goonjcustom cron.
+}
+
+/**
+ * Goonjcustom.CollectionCampCron API.
+ *
+ * @param array $params
+ *
+ * @return array API result descriptor
+ *
+ * @see civicrm_api3_create_success
+ * @see civicrm_api3_create_error
+ *
+ * @throws \CRM_Core_Exception
+ */
+function civicrm_api3_goonjcustom_institution_collection_camp_cron($params) {
+  $returnValues = [];
+  $optionValues = OptionValue::get(FALSE)
+    ->addWhere('option_group_id:name', '=', 'eck_sub_types')
+    ->addWhere('name', '=', 'Institution_Collection_Camp')
+    ->addWhere('grouping', '=', 'Collection_Camp')
+    ->setLimit(1)
```

Comment:
Why we need this

Reply:
To fetch the subtype of collection camp , which is `Institution_Collection_Camp` in my case